scripts/st2bootstrap-el6.sh

```diff
@@ -224,7 +224,7 @@ check_st2_host_dependencies() {
   VAR_SPACE=`df -Pk /var/lib | grep -vE '^Filesystem|tmpfs|cdrom' | awk '{print $4}'`
   if [ ${VAR_SPACE} -lt 358400 ]; then
     echo ""
```

Comment:
Should there be a mongodb change here?

Reply:
I'm leaving el6 alone. Next step before `st2 3.3.0` is to deprecate it and stop building packages.